### DISCUSSION TITLE
Integrity of ensemble with changes to model

### DISCUSSION THREAD

**Discussion by laat0003** [usgs/pestpp#239] [Q&A] [answered]
Hi,

I am after some guidance or information on the use of a pre-existing ensemble with changes to the model. I realize that this is undoubtedly dependent on the model and the proposed changes plus the modeling question but am nevertheless curious if there are any publications that have examined this in any capacity.
My specific quandary is related to model grid elevation changes in MF6 that will ultimately affect intercell conductance, albeit limited to a small area relative to the complete model domain. The existing ensemble is a GLM+IES developed stack with MOU forecast that has had some, shall we say, very experienced UA people involved. 

Cheers.

**Comment by jtwhite79** [comment, marked as answer]
I dont know of any pubs but its a scary topic.  @mnfienen used to say "everything is conditional" and it took me a while to realize what he was getting at:  those predictive results are implicitly "conditional" (in a bayesian sense) on all that model structure that underpins the analysis.  So if you change the model, it could produce (very) different results, still valid, but different.  I'd be curious to see what happens in your case with a limited change to layer elevation - if you were representing HK uncertainty spatially, then maybe you've already captured that potential conductance variation in the uncertainty analysis results (assuming this change does produce drying conditions or some other step-change)... Ive seen other anecdotal cases where changes in layer elevations made numerical zero difference in the results but this was for a water-balance model...I guess like you say, it depends on the prediction(s)...sorry that's not very helpful...

**Reply by laat0003** [reply]
Thanks @jtwhite79. These were my thoughts too, but I needed some confirmation - so still helpful. The changes in question need only be applied for the forecast period while the history match period would remain identical between the two models. For context, the model is for an open pit mine where the existing model assumes backfilling as part of the forecast while the new modelling question considers a pit lake. It is consequently expected that the forecasts will be different, but I wasn't 100% sure if, in this instance, the cell thickness change despite being limited to the forecast period, would potentially have any repercussions for the GLM+IES stack.
